### PR TITLE
YUPANA #58 - Add clowdapp for yupana

### PR DESCRIPTION
**Pull request by apuntamb**
Adding @san7ket into the loop.

### CODE REVIEW

**Comment by apuntamb** [comment]
@san7ket mind taking a look at this? if you could find some changes may be just by a glance?

File: clowdapp.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    database: #FIXME
+      name: yupana-db
+      version: 12
+    kafkaTopics:
+      - topicName: platform.upload.qpc
+        partitions: 1
+      - topicName: platform.inventory.host-ingress
+        partitions: 1
+      - topicName: platform.upload.validation
+        partitions: 1
```
Comment (apuntamb) [inline clowdapp.yaml:64]: @kgaikwad Any suggestions on these kafka topics? Are there more to be considered?

Comment (kgaikwad) [reply clowdapp.yaml:64]: Verified! Service uses only above topics.

**Comment by apuntamb** [comment]
Thanks for verifying this @kgaikwad ! Merging this.

**Review by kgaikwad — APPROVED** [review]
Looks good.
@san7ket - do you have any additional comments?

**Comment by apuntamb** [comment]
Thanks for reviewing @kgaikwad ! Merging this.

**Pull request merged**